recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+MKDIR build\windows
+CD build\windows
+
+cmake ^
+    -G "NMake Makefiles"                     ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%  ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX%     ^
+    -DCMAKE_BUILD_TYPE=Release               ^
+    ..\..
+
+cmake --build . --config Release
+
+dir
+dir libmetis
+dir programs
+dir include
+dir Release
+dir ..\..\include
+dir ..\..\
+dir ..\..\graphs
+
+copy libmetis\metis.lib %LIBRARY_LIB%
+copy programs\cmpfillin.exe %LIBRARY_BIN%
+copy programs\gpmetis.exe %LIBRARY_BIN%
+copy programs\graphchk.exe %LIBRARY_BIN%
+copy programs\m2gmetis.exe %LIBRARY_BIN%
+copy programs\ndmetis.exe %LIBRARY_BIN%
+copy programs\mpmetis.exe %LIBRARY_BIN%
+copy ..\..\include\metis.h %LIBRARY_INC%
+
+CD programs
+mpmetis.exe ..\..\..\graphs\metis.mesh 10

```

Comment:
Sorry if I made this unclear. `if errorlevel 1 exit 1` needs to be on a new line after each exe call.